rero_ils/theme/templates/security/email/reset_notice.html

```diff
@@ -0,0 +1,29 @@
+{# -*- coding: utf-8 -*-
+
+    RERO ILS
+    Copyright (C) 2021 RERO
+    Copyright (C) 2015-2018 CERN
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, version 3 of the License.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#}
+
+{%- if user.profile.first_name and user.profile.last_name %}
+<p>{{ _('Dear') }} {{ user.profile.first_name }} {{ user.profile.last_name }},</p>
+{%- else %}
+<p>{{ _('Dear patron') }},</p>
+{%- endif %}
+<p>{{ _('Your password has been successfully reset.') }}</p>
```

Comment:
TODO: in a futur, we need a message to explain what to do if the patron did not change his password. (Hacking)